README.md

```diff
@@ -1,4 +1,58 @@
+- Other topics will certainly be considered, but it should be something of interest to the Node.js community
+
+## Quality
+We are looking for posts that teach and provide value to our community. Contributions should include the meta-narrative that “Node.js is maturing and gaining widespread use in mainstream enterprises, while continuing  to inspire amazing potential and creativity among the massive community using the platform.” Sub-domains and topics for the Medium page include:
+
+- Serverless/Cloud
+- Containers
+- Community
+- Best Practices (How To)
+- Embedded/IoT
+- AI/Mobile
+- More (Essentially news from the Foundation and potentially more editorial style content about the future of Node.js Core) - see Google Cloud Platform’s More section as an example [here](https://cloudplatform.googleblog.com).
+
+## Guidelines/General Tips of Submissions
+
+- Posts from members and community members must be vendor neutral, though they may mention vendors involved in specific project deployments if this serves the purpose of educating readers. 
+- Posts about hosting of an in-person event or speaking at an event, or other indications of meaningful participation in the community, are allowed, but again shouldn’t feel like an advertisement for your services or company. 
+- Your post must be your original content, but can be published on the Node.js Foundation Medium publication, if you received permission from the editors of the website to do so.
+- All content should have a byline (preferably by a company engineer) and be published with Creative Commons with Attribution, so you’re welcome to republish on your blog.
+- The most interesting posts are ones that teach or explain how to do something in a way maybe others haven’t thought of. Good blog posts show hurdles that were encountered and explain how they were overcome (not that everything is rainbows and unicorns). 
+- A few posts that have performed well on the Node.js Foundation Medium page include Walmart Strives to be an Online Retailer with Node.js (use case); Node.js Transitions to LTS (news); The Progress of Node.js Post io.js and Node.js merger (progress update).
+- When showing upstreaming of a patch fixing an issue for others, link back to the Github issue so readers can follow along. 
+- We don’t avoid critical commentary or broad issues, but approach them with sensitivity, professionalism and tact in a way that is beneficial and positive for the community. 
+- Node.js Foundation Medium posts should be no longer than 1,000 words and no shorter than 300 words. 
+
+Diagrams or photos are strongly encouraged. 
+
+## How to submit for consideration
+
+- Please submit a brief summary and the topic of the post to `medium@nodejs.org` for consideration. The email alias reaches the review team consisting of Zibby Keaton, Sarah Conway, Tracy Hinds, Mikeal Rogers, Tierney Coren, Ross Kukulinski (with the potential to expand in the future).
```

Comment:
I think we can drop this list - it is already out of date. I think a simple way to keep the reference is to say that it goes to members of the Node.js Collection Team.